### PR TITLE
Design Tokens Manager - Adds "content sizing" to scale generator results

### PR DESCRIPTION
**Pull request by halocline**
[Deploy Preview](https://deploy-preview-INSERT_PR_#_HERE--keen-mayer-a86c8b.netlify.app/)

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

### CODE REVIEW

File: design-tokens-manager/src/App.jsx [hunk 1]
```diff
@@ -49,7 +54,7 @@ function App() {
             <Button
               icon={darkMode ? <Moon /> : <Sun />}
               onClick={() => setDarkMode(!darkMode)}
-              tip={`Swith to ${darkMode ? 'light' : 'dark'} mode`}
+              tip={`Switch to ${darkMode ? 'light' : 'dark'} mode`}
```
Comment (taysea) [inline design-tokens-manager/src/App.jsx:57]: Thank you.

**Comment by taysea** [comment]
Screenshot 2024-12-06 at 4 28 21 PM

It feels a little awkward how the "hide scale settings" and copy button wrap below

**Comment by taysea** [comment]
> Screenshot 2024-12-06 at 4 28 21 PM
> It feels a little awkward how the "hide scale settings" and copy button wrap below

Also, I'm assuming the content scale stopped at "xlarge" because I only could chose "20" steps for my primitive scale and there was nothing bigger to get to?

**Comment by halocline** [comment]
> It feels a little awkward how the "hide scale settings" and copy button wrap below

Fixed.

**Comment by halocline** [comment]
> Also, I'm assuming the content scale stopped at "xlarge" because I only could chose "20" steps for my primitive scale and there was nothing bigger to get to?

Yes, that is what was happening. --> I made a couple enhancements to:
1. Changed steps to be a RangeInput
2. Added more options to the number steps
3. Clear out the content sizes when a content base value is undefined

**Review by taysea — APPROVED** [review]
Refinements look good, I like the rangeinput.

**Pull request merged**